examples/seismic/elastic/operators.py

```diff
@@ -1,23 +1,80 @@
 
-from devito import Eq, Operator, TimeFunction, left, right, staggered_diff
+from devito import Eq, Operator, TimeFunction
 from examples.seismic import PointSource, Receiver
 
 
-def ForwardOperator(model, source, receiver, space_order=4,
-                    save=False, **kwargs):
+def stress_fields(model, save, space_order):
     """
-    Constructor method for the forward modelling operator in an elastic media
+    Create the TimeFunction objects for the stress fields in the elastic formulation
+    """
+    if model.grid.dim == 2:
+        stagg_xx = stagg_zz = (0, 0, 0)
```

Comment:
having to provide these 0-1 masks is very low level... can we not rather provide a tuple of dimensions, representing the staggered dimensions of the Function? then internally we translate this information into 0-1 bits. 
if we do that, perhaps to create these TimeFunctions we can getaway with a simple loop, rather than "if 1D do this, elif 2D do this, elif 3D do this, elif 4D do this, ..."